apps/api/src/app/auth/auth.controller.ts

```diff
@@ -59,13 +61,19 @@ export class AuthController {
   ) {}
 
   @Get('/github')
-  githubAuth() {
+  githubAuth(context: ExecutionContext) {
+    Logger.verbose('Checking Github Auth');
+    Logger.debug('Context: ' + context);
     if (!process.env.GITHUB_OAUTH_CLIENT_ID || !process.env.GITHUB_OAUTH_CLIENT_SECRET) {
-      throw new ApiException(
+      const err = new ApiException(
         'GitHub auth is not configured, please provide GITHUB_OAUTH_CLIENT_ID and GITHUB_OAUTH_CLIENT_SECRET as env variables'
       );
+      Logger.error(err);
+      throw err;
```

Comment:
@Cliftonz @scopsy 
Do we want to log all the exceptions that are thrown? if we do maybe it will be a good idea to place this log on nest ResponseInterceptor or ExceptionInterceptor. wdyt?

Reply:
I think logging exceptions in the middleweare/interceptor is a better approach. We can still log error when we are catching them and not rethrowing.